bin/mail/2016-08-09-hoc-latam/generate-mailing-lists.rb

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# Latin America (latam), 2 sub-lists for es & pt
```

Comment:
For someone who might not know what the codes mean, recommend:

`# Latin America (latam), 2 sub-lists for Spanish (es) speaking and Portuguese (pt) speaking countries